rules-tests/Php80/Rector/Property/NestedAnnotationToAttributeRector/Fixture/with_html.php.inc

```diff
@@ -0,0 +1,38 @@
+<html>
+    Some HTML content
+</html>
+
+<?php
+
+use Doctrine\ORM\Mapping\JoinColumn;
+use Doctrine\ORM\Mapping\JoinTable;
+
+final class WithHTML
+{
+    /**
+     * @JoinTable(name="lemma_type",
+     *      inverseJoinColumns={@JoinColumn(name="lemma_id", referencedColumnName="lemma_id")}
+     * )
+     */
+    private iterable $lemmas;
+}
+
+?>
+-----
+<html>
+    Some HTML content
+</html>
+
+<?php
+
+use Doctrine\ORM\Mapping\JoinColumn;
+use Doctrine\ORM\Mapping\JoinTable;
+
+final class WithHTML
+{
+    #[JoinTable(name: 'lemma_type')]
+    #[\Doctrine\ORM\Mapping\InverseJoinColumn(name: 'lemma_id', referencedColumnName: 'lemma_id')]
```

Comment:
this can't be added to use statement via `UseNodesToAddCollector` due to content has html 

https://github.com/rectorphp/rector-src/blob/cfebef69d9503d566029d590cd7d9425803ecd9e/rules/Php80/Rector/Property/NestedAnnotationToAttributeRector.php#L222